src/Raven.Server/NotificationCenter/BackgroundWork/DatabaseStatsSender.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Raven.Server.Documents;
+using Raven.Server.Documents.SqlReplication;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.NotificationCenter.BackgroundWork
+{
+    public class DatabaseStatsSender : BackgroundWorkBase
+    {
+        private readonly DocumentDatabase _database;
+        private readonly NotificationCenter _notificationCenter;
+
+        private Stats _latest;
+
+        public DatabaseStatsSender(DocumentDatabase database, NotificationCenter notificationCenter)
+            : base(database.Name, database.DatabaseShutdown)
+        {
+            _database = database;
+            _notificationCenter = notificationCenter;
+        }
+
+        protected override async Task Run()
+        {
+            while (CancellationToken.IsCancellationRequested == false)
+            {
+                try
+                {
+                    await Task.Delay(_notificationCenter.Options.DatabaseStatsThrottle, CancellationToken);
```

Comment:
Yes, every 5 seconds, but only if the studio is watching. Once we don't have any active watchers (studio) we stop this background work completely.